README.md

```diff
@@ -0,0 +1,313 @@
+# API Advisor
+
+```json
+"baseURL": "https://csgo-dashboard-rank.herokuapp.com"
+```
+---
+## Summary
```

Comment:
Some of the links in the summary are not working, i believe it has something related to the details tag nesting, we need to think some other approach to make this looks nicer

Reply:
done